docs/source/api/recipes/recipes.ipynb

```diff
@@ -85,7 +85,7 @@
    "outputs": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    avg_shortest_path = np.sum(lens) / (N * (N-1))
</pre><p>Where is N defined?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/513/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
I feel like the recipe should be self-contained.